packages/contracts/contracts/Unirep.sol

```diff
@@ -308,7 +309,6 @@ contract Unirep is IUnirep, VerifySignature {
         // Verify the proof
         if (!userStateTransitionVerifier.verifyProof(proof, publicSignals))
             revert InvalidProof();
-        if (publicSignals[5] >= type(uint160).max) revert AttesterInvalid();
```

Comment:
`AttesterInvalid` is removed here because `AttesterNotSignUp` will handle the same situation